### ISSUE TITLE
IPv6 audit of web based documentation

### ISSUE THREAD

**Issue by pmichali** [kubernetes/website#7566]
**This is a...** 

- [ ] Feature Request
- [X] Bug Report

**Problem:**
I did an audit of the website pages, looking for use of IPv4 and IPv6 in the pages and have info on the .md files examined. There are a few minor updates, which can be done (I can do, if you'd like).

Files with IPv4 mentioned that look good are:
```
cn/docs/concepts/cluster-administration/sysctl-cluster.md  OK
cn/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md  OK
cn/docs/getting-started-guides/coreos/bare_metal_offline.md  OK
docs/concepts/cluster-administration/networking.md  OK future could give IPv6 examples and/or DinD
docs/concepts/cluster-administration/sysctl-cluster.md  OK
docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md  OK
docs/getting-started-guides/coreos/bare_metal_offline.md  OK
docs/getting-started-guides/kubespray.md  OK
docs/setup/independent/create-cluster-kubeadm.md  OK
docs/setup/independent/high-availability.md  OK
docs/tasks/administer-cluster/ip-masq-agent.md  OK
```

Files with IPv6 mentioned that look good are:
```
cn/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md  OK
cn/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md  OK
docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md  OK
docs/getting-started-guides/ubuntu/local.md  ?? Says IPv6 not supported for Juju/LXD. Likely is correct.
docs/reference/setup-tools/kubeadm/implementation-details.md  OK
docs/setup/independent/create-cluster-kubeadm.md  OK
docs/tasks/access-application-cluster/access-cluster.md  OK
```

**Proposed Solution:**
Files with IPv4 mention that could use changes:
```
docs/getting-started-guides/scratch.md  !! Mentions IPv6 not support, talks about IPv4 (could say using Ipv4 as example)
```

Files with IPv6 mention that could use changes:
```
docs/concepts/services-networking/service.md  ?? Related to load balancers, says IPv6 not supported for source ranges. I think we can remove this.
docs/getting-started-guides/scratch.md  !! Says IPv6 not supported for pod IPs. Can be removed.
docs/imported/release/notes.md  !! Indicates /66 limitation, which has just been removed. Should indicate limitation for 1.9 and removal for 1.10.
```

**Page to Update:**
See above.

**Comment by pmichali** [comment]
Just pulled latest from master today, and many of the .md changes are no longer there. Right now, I can only see one .md file that needs changes, docs/getting-started-guides/scratch.md.  There are three .html files that could be changed, but I'm not sure if two of them are older releases (and hence should be left alone).

**Comment by zacharysarah** [comment]
Closed by #7606.
